### DISCUSSION TITLE
link block on a column?

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1491] [Q&A]
Has anyone found a way to wrap a column in a link block? If you have, I'd love to know what explicit steps to do so.

**Comment by meetmandeep** [comment]
Wrapping a column in a link block is not possible nor recommended. Why not add a link block to the column and add a section inside it with other content/blocks?

**Reply by eswanzey** [reply]
Yes, maybe it's a bug but I'm not finding a way to vertically center a section within the column whereby I can have it cover the column vertically 100% and center vertically. Removing the section and just using the base element (a header), doesn't allow for vertical centering either. Put another way, I'm not seeing a way to vertically center a section or header within a column, even though the column is set to vertical-middle and margin/padding is applied.

**Reply by meetmandeep** [reply]
I'm not a CSS expert but it looks like what you're trying to achieve can be done with Flex. Modify the Section styling from Styling Tab and change the Display to flex; experiment with direction, align, and justify. 

Official release has better labels explaining these functions. You can see my experimental settings in the screenshot below to vertically align a section in column.

![image](https://user-images.githubusercontent.com/29958123/123876962-28e8de80-d8f1-11eb-9bc7-0b38543be55e.png)

**Reply by eswanzey** [reply]
I would invite anyone to try and do the following using just blocks and settings, with no extraneous scripts, then publish the steps if they succeed.

The challenge: have a multi-column grid, and in each column, embed a full-height, full-width linked section that contains a full-width text block that is centered within the section.
